projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -184,6 +196,12 @@ class_exists( 'Jetpack_AMP_Support' )
 			// Override Notification module to include RTL styles.
 			add_action( 'a8c_wpcom_masterbar_enqueue_rtl_notification_styles', '__return_true' );
 		}
+
+		// Hides and replaces the language dropdown for the current user.
+		if ( IS_PROFILE_PAGE ) {
```

Comment:
`IS_PROFILE_PAGE` looks like it should only be true when you're editing your own profile:
`define( 'IS_PROFILE_PAGE', ( $user_id == $current_user->ID ) );`

But I'm still seeing this change when I edit another user's profile. 🤔

Reply:
Looks like it's not defined here, so we may need another approach for checking that this is the current user editing themselves, e.g. check the `user_id` query param.